homeassistant/components/evohome/__init__.py

```diff
@@ -153,14 +144,14 @@ class EvoDevice(Entity):
     (optionally) a DHW controller.
     """
 
-    def __init__(self, evo_data, client, obj_ref):
+    def __init__(self, evo_data, client, evo_device_ref):
         """Initialize the evohome entity."""
         self._client = client
-        self._obj = obj_ref
+        self._evo_ref = evo_device_ref
```

Comment:
I think `_ref` is redundant. I suggest:
```suggestion
        self._evo_device = evo_device
```